api/scpca_portal/models/job.py

```diff
@@ -112,9 +114,18 @@ def _batch(self):
         """
         return boto3.client("batch", region_name=settings.AWS_REGION)
 
+    @property
+    def _logger(self):
```

Comment:
Handle logger the same way as `models/contact.py` no need to bring it in as a model property.